docs/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -596,9 +610,9 @@ module.exports = withSentryConfig(nextConfig, {
 
 ### Opt Out of Sentry SDK bundling in Client or Server side
 
-If you want the `sentry` to be available in your server side & not in client side, you can make your `sentry.client.config.js` empty. This will prevent webpack from pulling in the Sentry related files when generating the browser bundle. The same goes the opposite for opting out of server side bundle by not initializing Sentry inside `instrumentation.(js|ts)`.
+If you want the `sentry` to be available in your server side & not in client side, you can make your `sentry.client.config.js` empty. This will prevent webpack from pulling in the Sentry related files when generating the browser bundle. The same goes the opposite for opting out of server side bundle by emptying `sentry.server.config.js`.
```

Comment:
```suggestion
If you want the Sentry SDK to be available server-side & not client-side, you can make your `sentry.client.config.js` empty. This will prevent webpack from pulling in the Sentry-related files when generating the browser bundle. Similarly, to opt out of server-side SDK bundling, make your `sentry.server.config.js` empty.
```